docs/hub/keras.md

```diff
@@ -1,75 +1,54 @@
 
-`keras` is an open-source machine learning library that uses a consistent and simple API to build models leveraging TensorFlow and its ecosystem.
+Keras is an open-source multi-backend deep learning framework, with support for JAX, TensorFlow, and PyTorch. You can find more details about it on [keras.io](https://keras.io/).
 
 ## Exploring Keras in the Hub
 
 You can find over 200 `keras` models by filtering at the left of the [models page](https://huggingface.co/models?library=keras&sort=downloads).
 
-All models on the Hub come up with useful feature:
-1. An automatically generated model card with a description, a plot of the model, and more.
-2. Metadata tags that help for discoverability and contain information such as license.
-3. If provided by the model owner, TensorBoard logs are hosted on the Keras repositories.
+Keras models on the Hub come up with useful features:
+1. A generated model card with a description, a plot of the model, and more.
+2. A download count to monitor the popularity of a model.
+3. A code snippet to quickly get started with the model.
 
 
 ## Using existing models
 
-The `huggingface_hub` library is a lightweight Python client with utility functions to download models from the Hub.
+Keras is deeply integrated with the Hugging Face Hub. This means you can load and save models on the Hub directly from the library. To do that, you need to install a recent version of Keras and `huggingface_hub`. The `huggingface_hub` library is a lightweight Python client used by Keras to interact with the Hub.
 
 ```
-pip install huggingface_hub["tensorflow"]
+pip install -U keras huggingface_hub
 ```
 
-Once you have the library installed, you just need to use the `from_pretrained_keras` method. Read more about `from_pretrained_keras` [here](https://huggingface.co/docs/huggingface_hub/main/en/package_reference/mixins#huggingface_hub.from_pretrained_keras).
+Once you have the library installed, you just need to use the regular `keras.saving.load_model` method by passing as argument an HF path. An HF path is a `repo_id` prefixed by `hf://` e.g. `"hf://keras-io/weather-prediction"`. Read more about `load_model` in [Keras documentation](https://keras.io/api/models/model_saving_apis/model_saving_and_loading/#load_model-function).
 
 ```py
-from huggingface_hub import from_pretrained_keras
+import keras
 
-model = from_pretrained_keras("keras-io/mobile-vit-xxs")
-prediction = model.predict(image)
-prediction = tf.squeeze(tf.round(prediction))
-print(f'The image is a {classes[(np.argmax(prediction))]}!')
-
-# The image is a sunflower!
+model = keras.saving.load_model("hf://Wauplin/mnist_example")
 ```
 
-If you want to see how to load a specific model, you can click **Use in keras** and you will be given a working snippet that you can load it! 
-
-<div class="flex justify-center">
-<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-keras_snippet1.png"/>
```

Comment:
keep some version of the existing one in a different (`tf-keras`?) page?

Reply:
right, I buried tf-keras a bit too quickly. I added some docs back in https://github.com/huggingface/hub-docs/pull/1320/commits/b9d3fb9cef4fb517135f94ec65d3eb650a5b076c and https://github.com/huggingface/hub-docs/pull/1320/commits/8146d787bfc72e0e09813fab9ed7518d0313baa6 as a section at the end of the Keras.md page, specially mentioning it's legacy and recommended to upload Keras3 models for better reach.

Reply:
i would have kept a separate page (if only because the library ids are ≠), but 🤷

Reply:
Make sense if we want to reference it. I moved it to a new TF-Keras page in https://github.com/huggingface/hub-docs/pull/1320/commits/f4707f44e67f50f987db0e3d446cf9d56d207236 and https://github.com/huggingface/hub-docs/pull/1320/commits/9df147e873f3db8bdb87cd86b13250fe511e278e.